src/main/java/com/fryrank/controller/ReviewController.java

```diff
@@ -13,12 +13,12 @@
 import org.springframework.validation.BindingResult;
 import org.springframework.web.bind.annotation.*;
 
-import static com.fryrank.Constants.API_PATH;
-
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
+import static com.fryrank.Constants.*;
```

Comment:
Good practice to not use star imports because it could result in name conflicts and in general imports more than is needed